pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -204,6 +205,15 @@ type ACMEIssuerDNS01ProviderAcmeDNS struct {
 	AccountSecret SecretKeySelector `json:"accountSecretRef"`
 }
 
+// ACMEIssuerDNS01ProviderDNSMadeEasy is a structure containing the
+// configuration for DNSMadeEasy
+type ACMEIssuerDNS01ProviderDNSMadeEasy struct {
+	BaseURL   string            `json:"baseURL"`
+	APIKey    string            `json:"apiKey"`
+	SecretKey SecretKeySelector `json:"secretKeySecretRef"`
```

Comment:
What's the difference between apiKey and secretKey? APIKey sounds like a 'sensitive' bit of info - should it be a SecretKeySelector?

Reply:
They're analogous to an AWS access key (apiKey) and secret key pair (secretKey).